kubectl-minio/cmd/tenant-create.go

```diff
@@ -81,10 +83,20 @@ func newTenantCreateCmd(out io.Writer, errOut io.Writer) *cobra.Command {
 	return cmd
 }
 
-func (c *createCmd) validate() error {
+func (c *createCmd) validate(args []string) error {
+	c.tenantOpts.Name = args[0]
 	c.tenantOpts.SecretName = c.tenantOpts.Name + tenantSecretSuffix
 	c.tenantOpts.ConsoleSecret = c.tenantOpts.Name + consoleSecretSuffix
 	c.tenantOpts.Image = helpers.DefaultTenantImage
+	if args == nil {
+		return errors.New("provide the name of the tenant, e.g. 'kubectl minio tenant create tenant1'")
```

Comment:
"This command requires specifying the tenant name as an argument, e.g. <example"

Reply:
If there are no objects to this structure, please apply throughout.